vue-app/src/store/actions.ts

```diff
@@ -92,8 +93,11 @@ const actions = {
     //TODO: update to take factory address as a parameter, default to env. variable
     const round = await getRoundInfo(roundAddress)
     commit(SET_CURRENT_ROUND, round)
-    if (round && round.status === RoundStatus.Finalized) {
-      const tally = await getTally(roundAddress)
+  },
+  async [LOAD_TALLY]({ commit, state }) {
```

Comment:
We were fetching the tally file from the `LOAD_ROUND_INFO` action. Now, it has its own action which we call from the Claim button if necessary.

Reply:
Confirm this was tested, otherwise lgtm!

Reply:
Yes, it was tested.

The only place where we are using the tally file is in the Claim button component context. What I mean is that there are not a lot of places where this change can have an impact, just one.